packages/project/src/v4-specification.ts

```diff
@@ -1147,6 +1164,7 @@ export interface Field {
  *  "crops": [
  *    // ... crops that were planted in year 2000
  *  ],
+ *  "dataTimeFrame": GROWER_REPORTED,
```

Comment:
should this be the following (I don't know the answer)
```suggestion
 *  "dataTimeFrame": "GROWER_REPORTED",
```